pkgs/os-specific/linux/esdm/default.nix

```diff
@@ -0,0 +1,78 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, protobufc
+, pkg-config
+, fuse3
+, meson
+, ninja
+, libselinux
+, jitterentropy
+, selinux ? false
+, drng_hash_drbg ? true
+, drng_chacha20 ? false
+, ais2031 ? true
+, linux-devfiles ? true
+, linux-getrandom ? true
+, es_jitterRng ? true
+, es_cpu ? true
+, es_kernel ? true
+, es_irq ? true
+, es_sched ? true
+, es_hwrand ? true
+, hash_sha512 ? false
+, hash_sha3_512 ? true
```

Comment:
Why do we need all those options?

Reply:
Im going to add comments to these, to explain them a bit. The comments are added with commit: (https://github.com/NixOS/nixpkgs/pull/238777/commits/8be5b2fe775d868b1c853f97d5f2e744cd219380)